tests/ops/integration_tests/saas/test_aircall_task.py

```diff
@@ -0,0 +1,230 @@
+        v[f"{dataset_name}:contact"],
+        min_size=1,
+        keys=[
+            "id",
+            "first_name",
+            "last_name",
+            "company_name",
+            "information",
+            "is_shared",
+            "direct_link",
+            "created_at",
+            "updated_at",
+            "phone_numbers",
+            "emails",
+        ],
+    )
+
+    assert_rows_match(
+        v[f"{dataset_name}:calls"],
+        min_size=1,
+        keys=[
+            "id",
+            "direct_link",
+            "direction",
+            "status",
+            "missed_call_reason",
+            "started_at",
+            "answered_at",
+            "ended_at",
+            "duration",
+            "voicemail",
+            "recording",
+            "asset",
+            "raw_digits",
+            "user",
+            "contact",
+            "archived",
+            "assigned_to",
+            "tags",
+            "transferred_by",
+            "transferred_to",
+            "teams",
+            "number",
+            "cost",
+            "country_code_a2",
+            "pricing_type",
+            "comments"
+        ],
+    )
+    # verify we only returned data for our identity email
+    assert v[f"{dataset_name}:contact"][0]["phone_numbers"][0]["value"] == aircall_identity_phone_number
+
+
+@pytest.mark.integration_saas
+@pytest.mark.integration_aircall
+@pytest.mark.asyncio
+async def test_aircall_erasure_request_task(
+    db,
+    policy,
+    erasure_policy_string_rewrite,
+    aircall_connection_config,
+    aircall_dataset_config,
+    aircall_erasure_identity_phone_number,
+    aircall_erasure_identity_email,
+    aircall_create_erasure_data,
+) -> None:
+    """Full erasure request based on the aircall SaaS config"""
+
+    masking_strict = CONFIG.execution.masking_strict
+    CONFIG.execution.masking_strict = False  # Allow Delete
+
+    privacy_request = PrivacyRequest(
+        id=f"test_aircall_erasure_request_task_{random.randint(0, 1000)}"
+    )
+    identity = Identity(**{"phone_number": aircall_erasure_identity_phone_number})
+    privacy_request.cache_identity(identity)
+
+    dataset_name = aircall_connection_config.get_saas_config().fides_key
+    merged_graph = aircall_dataset_config.get_graph()
+    graph = DatasetGraph(merged_graph)
+
+    v = await graph_task.run_access_request(
+        privacy_request,
+        policy,
+        graph,
+        [aircall_connection_config],
+        {"phone_number": aircall_erasure_identity_phone_number},
+        db,
+    )
+
+    assert_rows_match(
+        v[f"{dataset_name}:contact"],
+        min_size=1,
+        keys=[
+            "id",
+            "first_name",
+            "last_name",
+            "company_name",
+            "information",
+            "is_shared",
+            "direct_link",
+            "created_at",
+            "updated_at",
+            "phone_numbers",
+            "emails",
+        ],
+    )
+
+    # assert_rows_match(
```

Comment:
Don't leave commented code in the final version

Reply:
removed the commented code